conda.el

```diff
@@ -336,6 +352,46 @@ It's platform specific in that it uses the platform's native path separator."
     (if (or conda-message-on-environment-switch (called-interactively-p 'interactive))
 	(message "Switched to conda environment: %s" env-name))))
 
+;;;###autoload
+(defun conda-env-activate-path (&optional path)
+  "Switch to environment PATH, prompting if called interactively."
+  (interactive)
+  (let ((env-path (or path (read-directory-name "choose directory of conda env: "))))
```

Comment:
Can we change the prompt here to `"Conda environment directory:"`?

Reply:
:+1: